submissions/NadiaVorontsova/hooli-style-popup/index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>HTML-CSS-Popup</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="header__menu">
+                <ul class="header__list">
+                    <li><a href="#" class="header__link link">Mail</a></li>
+                    <li><a href="#" class="header__link link">Images</a></li>
+                    <li>
+                        <input type="checkbox" id="popup_checkbox" class="header__popup__checkbox" checked>
+                        <label for="popup_checkbox" class="header__popup__label">
+                            <img src="assets/popup-button.png" width="25" height="25" alt="" tabindex="0"
+                                class="header__popup__btn">
+                        </label>
+                        <div class="header__popup__menu">
+                            <ul class="header__popup__menu__items">
+                                <li class="header__popup__menu__item">
+                                    <a href="#" class="header__popup__menu__link link">
+                                        <div class="header__popup__menu__container">
+                                            <img src="assets/calendar.png" width="48" height="35" alt=""
+                                                class="header__popup__menu__img">
```

Comment:
Those class names are too long, try to use [BEM](https://en.bem.info/methodology/)

Reply:
I tried to change class name